### ISSUE TITLE
AP course fixes

### ISSUE THREAD

**Issue by MathTauAthogen** [menai/PowerGPA#74]
There is a problem where, in my school, only a certain number of AP courses also count for credit, different each year. These limits aren't factored in.

**Comment by Nikolay-Pomytkin** [comment]
Yeah I'm aware of this. As far as I know, a max of 2 count for your sophomore year GPA, 3 for your junior year GPA, and 4 for senior year (let me know if that's incorrect). Although I'm not currently working on this project, I'm considering updating it in my free time, and I'll surely include this feature (I'll have to find each students graduation year in the API - assign them a grade and recalculate based on number of AP courses being taken). I'll close this issue when that process is complete.
